osbs/api.py

```diff
@@ -713,6 +713,47 @@ def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missin
 
         return logs
 
+    def _parse_build_log_entry(self, entry):
+        items = entry.split()
+        if len(items) < 4:
+            logger.warning("This is not a valid build log entry: %s", entry)
+            return (None, entry)
+        date = items[0]
+        time = items[1]
+        platform = items[2]
+        plen = sum(len(field) + 1  # include trailing space
+                   for field in [date, time, platform])
+        if not platform.startswith("platform:"):
+            logger.warning("Line logged without using the appropriate LoggerAdapter")
+            platform = None
```

Comment:
Side note: I'm not quite sure how to verify in the self-test that it actually does trip that warning, other than to cause a failure and look at the py.test debug and warning messages near the failure (which are indeed showing that line tripped w/my latest addition).